nginx/examples.conf

```diff
@@ -44,14 +39,9 @@
 
 ###
 ## Disable caching completely when developing the site
```

Comment:
Maybe we should mention here that having these enabled in production will decrease site performance significantly? Current wording might be unclear, ie. this could be interpreted in two ways:

> set these and there is no cache when developing

vs.

> when doing dev, set these to disable caching

Reply:
Good point!

Reply:
done